### PR TITLE
[#67] Correct implementation of postscript qcurve

### PR DESCRIPTION
**Pull request by hangpark**
See #67 for details.

### CODE REVIEW

File: app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java [hunk 1]
```diff
@@ -54,60 +51,81 @@ protected HangulCharacter() {
...
+          skeletonsData.get(i).add(joint);
+        }
+      }
+    }
+
+    // Copy skeletons data
     skeletons = new ArrayList<>();
     for (int i = 0; i < skeletonsData.size(); i++) {
       skeletons.add(i, new ArrayList<BezierCurve>());
+      Alert.log(this, "====");
```
Comment (bimak-server) [inline app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java]: remove alert

File: app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java [hunk 2]
```diff
@@ -54,60 +51,81 @@ protected HangulCharacter() {
...
       for (int j = 0; j < skeletonsData.get(i).size(); j++) {
+        Alert.log(this, skeletonsData.get(i).get(j).toString());
```
Comment (bimak-server) [inline app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java]: remove alert

**Review by bimak-server — CHANGES_REQUESTED** [review]
remove alert

File: app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java (hunk 1 above)
Comment (hangpark) [reply app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java]: sorry :)

**Review by xeoh — APPROVED** [review]
👍

**Review by bimak-server — APPROVED** [review]
👍

**Pull request merged**